tests/runtime/map

```diff
@@ -7,21 +7,17 @@ NAME map scalar
 PROG BEGIN { @a = 0; exit(); }
 EXPECT @a: 0
 
-NAME map auto scalar
+NAME map no auto scalar
```

Comment:
should we just get rid of these 'auto scalar' runtime tests now that nothing is being auto-scalared

Reply:
I've left them in place but made them a bit saner. I do think they are good rails with respect to the intended behavior either way.

Reply:
(They no longer say "auto scalar", they just say "scalar-like")